ulong_extras/gcd.c

```diff
@@ -1,6 +1,5 @@
 /*
-    Copyright (C) 2009, 2015 William Hart
-    Copyright (C) 2014 Abhinav Baid
+    Copyright (C) 2021 Albin Ahlbäck
```

Comment:
Since the function got a complete makeover, I thought it was best to remove the previous names. Let me know if I should change it back.